src/types/TabView.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { Dimensions, SafeAreaView } from 'react-native'
+import { TabView, SceneMap } from 'react-native-tab-view'
+
+export default class TabNavigation extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      index: 0,
+      routes: props.scenes,
+    }
+  }
+
+  render() {
+    const sceneMapConfig = this.props.scenes.reduce(
+      (config, scene) => ({ ...config, [scene.key]: scene.render }),
+      {},
+    )
+    return (
+      <TabView
+        navigationState={this.state}
+        renderScene={SceneMap(sceneMapConfig)}
```

Comment:
why not with bindings?

Reply:
really don't know 🤔 got it from last

Reply:
just put here the files needed to inside app navigation

Reply:
hmm okay, can change later for now it's okay